test/models/comment_test.rb

```diff
@@ -1,4 +1,4 @@
-require "test_helper"
+require 'test_helper'
```

Comment:
Great job done so far @julzedz 
Please permit me to use this line to request changes to this Pull Request since your code dose not reflect on the PR.

- [ ] Could you fix the error on line 5 in your comment migration file?  is `t.references :post, null: false, foreign_key: true`.

**Not belongs_to**
<img width="615" alt="Screenshot 2023-07-14 at 04 27 38" src="https://github.com/julzedz/tech-krunch/assets/55596970/e2102a48-6209-4b31-a2c1-7727c709dbc5">